libgap.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: @PACKAGE_NAME@
+Description: C/C++ library powering GAP, a computer algebra system
```

Comment:
The C++ is only on the inside, so I wouldn't mention. Actually, why mention C/C++ at all? Is that done for all libraries?!

Super nitpick: I am also not sure about the "powering GAP" part -- on the one hand, GAP doesn't use this at all; on the other, it is 99.9% of the GAP kernel.

Perhaps like this?
```suggestion
Description: Library containing the kernel of GAP, a computer algebra system
```